### DISCUSSION TITLE
failed to verify verification code

### DISCUSSION THREAD

**Discussion by 123flodirk** [icloud-photos-downloader/icloud_photos_downloader#966] [Q&A]
2FA gives the 6 figure  code on my iphone  for verificartion but after inputting  it  in my terminal I get " failed to verify verification code" 
I must be doing something wrong obviously...? I tried formats:
'823 736' or 823736 or "823736" or "823 736" ?  
big thank you for pointing me to the solution!

![image](https://github.com/user-attachments/assets/517e8929-a5ce-44e6-91a2-66ada8c0526a)

**Comment by 123flodirk** [comment]
to answer my own question the format should be:
823736

-just type the 6 numbers  without spaces

**Comment by AndreyNikiforov** [comment]
Can you specify cmd line you are using (replace sensitive info of course), pls? I do not recognize the prompt... Should be just 6 digits (without quotes or anything)

**Reply by 123flodirk** [reply]
Yes, just  6 digits works :) I think I was to quick or not focused enough (…)  BTW Thank you for replying , it is a great program !
